lua/dap.lua

```diff
@@ -393,8 +393,32 @@ end
 function M.continue()
   if not session then
     select_config_and_run()
-  else
+  elseif session.stopped_thread_id then
     session:_step('continue')
+  else
+    local prompt = (session.initialized
+      and "Session active, but not stopped at breakpoint> "
+      or "Session still initializing> "
+    )
+    local choices = {
+      {
+        label = "Stop session",
+        action = M.stop
+      },
+      {
+        label = "Restart session",
+        action = M.restart,
+      },
+      {
+        label = "Do nothing",
+        action = function() end,
+      },
+    }
```

Comment:
maybe even add alternative `pause`? In vscode continue and pause are the same button that toggles depending on current session state

Reply:
Sounds good. I added it